frontend/app/templates/components/home-menu.hbs

```diff
@@ -0,0 +1,22 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed"
+      data-toggle="collapse" data-target="#menu-collapse">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      {{ link-to 'Home' 'index' class="navbar-brand" }}
```

Comment:
I think you're following a very old guide/doc for ember. This syntax of link-to is deprecated. You should use the following.

```hbs
{{#link-to 'index' class='navbar-brand'}}
  Home
{{/link-to}}
```

Change all occurrences please :)